### ISSUE TITLE
receiving a message with an email duplicates content before the mail

### ISSUE THREAD

**Issue by angelo-fuchs** [deltachat/deltachat-desktop#609]
On: Ubuntu Bionic
![screenshot_delta_desktop_contact](https://user-images.githubusercontent.com/10619521/51480463-685e8880-1d91-11e9-82f7-c5ef034f1363.png)

The Name is shown twice.

To reproduce: Open Deltachat on different system (e.G. Android) and attach a Contact to a chat.

**Comment by Simon-Laux** [comment]
How did you send the contact? I can't reproduce this behavior currently.

**Comment by Simon-Laux** [comment]
Its probably in this file: `src/renderer/components/conversations/EmbeddedContact.js`
But I need to know how to send contacts in a way that triggers this.

**Comment by angelo-fuchs** [comment]
I chatted from my Delta Android to Delta Desktop.
Attach -> Contact.

**Comment by r10s** [comment]
just tested this, for me this works :)

in fact, the "send contacts" are just plain texts, two lines, name+address, there is nothing special with them on the receiving site.

**Comment by angelo-fuchs** [comment]
@r10s No. It is Desktop, just tested it again.

**Comment by angelo-fuchs** [comment]
(I didn't update Desktop before, so might be that it was solved already).

**Comment by angelo-fuchs** [comment]
Desktop Version 0.98.2-PREVIEW

**Comment by angelo-fuchs** [comment]
@r10s I can report that the text that was sent is exactly as you describe it: Just a name, a newline and an e-mail. Still Desktop displays it as shown in the screenshot above.

**Comment by hpk42** [comment]
On Sun, Feb 10, 2019 at 10:27 -0800, Angelo Fuchs wrote:
> @r10s I can report that the text that was sent is exactly as you describe it: Just a name, a newline and an e-mail. Still Desktop displays it as shown in the screenshot above.

seems like desktop is haunted with "double" showing of text.

**Comment by angelo-fuchs** [comment]
It is super weird. It is triggered by the combination one line, one newline, one line with an email

Triggers
```
Bla
bla@example.com
```
Doesn't trigger:
```
Bla
Blubb
```
Surprisingly also doesn't trigger:
```
Bla
bla@bla.bla
```

It seems that when there is an email, the sections above are displayed twice so:
```
Bla
Blubb
bla@example.com
```
gets displayed as:
```
Bla
Blubb
Bla
Blubb
bla@example.com
```

**Comment by angelo-fuchs** [comment]
It is multipliedd also by the amount of emails present so
```
One
Two
Three
Bla@example.com
Blubb@example.com
```
gets displayed as
```
One
Two
Three
One
Two
Three
Bla@example.com
Blubb@example.com
One
Two
Three
Bla@example.com
Blubb@example.com
```

**Comment by angelo-fuchs** [comment]
I just noticed that the newline is not needed as well.
```
Foo bla@example.com
```
gets displayed as
```
Foo Foo bla@example.com
```

**Comment by hpk42** [comment]
On Mon, Feb 11, 2019 at 03:52 -0800, Angelo Fuchs wrote:
> I just noticed that the newline is not needed as well.
> ```
> Foo bla@example.com
> ```
> gets displayed as
> ```
> Foo Foo bla@example.com
> ```

For the record, there were earlier problems when mail addresses 
are contained in messages -- they would double the whole message.
Is this maybe another case of parsing e-mail addresses interfering
with the rendering?

**Comment by Simon-Laux** [comment]
@angelo-fuchs was this solved with #617 ?  Does it happen in an recent version, too?

**Comment by Simon-Laux** [comment]
I'll close this now, as I'm 99% sure that we solved it with #617.
